src/videojs-contrib-hls.js

```diff
@@ -279,12 +279,15 @@ class HlsHandler extends Component {
     // start playlist selection at a reasonable bandwidth for
     // broadband internet (0.5 MB/s) or mobile (0.0625 MB/s)
     if (typeof this.options_.bandwidth !== 'number') {
-      // only use Android for mobile because iOS does not support MSE (and uses
-      // native HLS)
-      this.options_.bandwidth =
-        videojs.browser.IS_ANDROID ? INITIAL_BANDWIDTH_MOBILE : INITIAL_BANDWIDTH_DESKTOP;
+      this.options_.bandwidth = INITIAL_BANDWIDTH;
     }
 
+    // If the bandwidth number is unchanged from the initial setting
+    // then this takes precedence over the enableLowInitialPlaylist option
+    this.options_.enableLowInitialPlaylist =
+      (this.options_.enableLowInitialPlaylist &&
+       this.options_.bandwidth === INITIAL_BANDWIDTH) || false;
```

Comment:
the `|| false` is not necessary `a && b` will just be the value of `b` when `a == true`